ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@ const Yaml = (props: IProps) => {
       setStreaming(false)
     }
   }
+  const contentToTopHeight = contentRef.current?.getBoundingClientRect()?.top
```

Comment:
Potential null reference - consider adding a null check for `interpretEndRef.current` before accessing `getBoundingClientRect()`.